en/sim_gazebo_gz/README.md

```diff
@@ -31,16 +31,42 @@ sudo apt-get install gz-garden
 
 ## Running the Simulation
 
-Gazebo SITL simulation can be conveniently run through a `make` command as shown below:
+Gazebo SITL simulation can be conveniently run using a `make` command as shown below:
 
 ```sh
 cd /path/to/PX4-Autopilot
 make px4_sitl gz_x500
 ```
 
-This will run both the PX4 SITL instance and the Gazebo client.
+This runs both the PX4 SITL instance and the Gazebo client.
 Note that all gazebo make targets have the prefix `gz_`.
 
+Another way that Gazebo SITL can be connected is in _standalone mode_.
+In standalone mode PX4 SITL and Gazebo are started separately in their own terminals.
+You start PX4 in standalone mode by prefixing the make command with `GZ_PX4_STANDALONE=1`:
```

Comment:
So to be pedantic, prefixing the command with `GZ_PX4_STANDALONE` tells PX4 to start without also starting gz-server right? However it is still expecting GZ to start right? So mostly this is so you can have PX4 and GZ running in separate places? (again, why?)

Reply:
Yes. It will start PX4 without starting gz-server. You don't have to start GZ, but PX4 will not be usable in that case. It will just print the warning message over and over again.